index.css

```diff
@@ -28,13 +28,16 @@ h1, h2, h3, h4, h5, h6 {
     font-family: 'Raleway', sans-serif;
 }
 
+h1 {
+    font-size: 28px
+}
+
 .panel {
     width: 50%;
 }
 
 .section {
-    margin: 30px 0px 30px;
-    padding: 5px;
+    padding: 0.3em;
 }
 
 a:link {
```

Comment:
I think we also need to add another selector so that visited links do not turn purple.